src/witness/svcompSpec.ml

```diff
@@ -42,5 +45,6 @@ let to_string spec =
     | UnreachCall f -> "call(" ^ f ^ "())"
     | NoDataRace -> "data-race"
     | NoOverflow -> "overflow"
+    | Termination -> "no-termination"
```

Comment:
For termination, the `to_string` should be `CHECK( init(main()), LTL(F end) )`, see [comment above](  https://github.com/goblint/analyzer/pull/1093/files#r1307143458)